client/src/components/Game/Timer.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { withFirebase } from '../Firebase';
+import { withRouter } from 'react-router';
+import { compose } from 'recompose';
+
+class Timer extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      minutes: 99,
+      seconds: 99,
+      initialSet: false
+    }
+  }
+
+  async componentDidMount() {
+    const { id } = this.props.match.params;
+    const game = this.props.firebase.game(id);
+
+    game.get()
+    .then(docSnapshot => {
+      if(docSnapshot.exists) {
+        game.onSnapshot((snapshot) => {
+          if(snapshot.data().gameStartTime !== null) {
+            this.setState({
+              currentPlayerIndex: snapshot.data().players.length,
+              startTime: snapshot.data().gameStartTime.seconds,
+              timePerTurnInSeconds: parseInt(snapshot.data().timeLimit, 10) * 60,
+            })
+            this.setState({
+              timeTurnWillEnd: ((this.state.timePerTurnInSeconds * this.state.currentPlayerIndex) + this.state.startTime),
+              currentTime: Math.floor(new Date().getTime() / 1000),
+            })
+            this.setState({
+              minutes: Math.floor((this.state.timeTurnWillEnd - this.state.currentTime) / 60),
```

Comment:
Wait so if each player has 3 days, they'll have 4320 minutes displayed? It seems not too user friendly